src/main/kotlin/no/nav/syfo/kafka/recordprocessors/SpleisRecordProcessor.kt

```diff
@@ -31,6 +31,10 @@ class SpleisRecordProcessor {
     }
 
     private fun processUtbetalingSpleisEvent(utbetaling: UtbetalingSpleis) {
-        utbetalingSpleisDAO.storeSpleisUtbetaling(utbetaling)
+        if (!utbetalingSpleisDAO.isSpleisUtbetalingExists(utbetaling.fødselsnummer, utbetaling.utbetalingId)) {
```

Comment:
Foreslår kanskje å droppe sjekken fra spleis, siden det ikke har skjedd her. Spleis har heller ikke samme unique constraint - ser ut som denne kun har contraint på UTBETALING_ID

Reply:
Jeg har slettet sjekken for spleis, men jeg vill beholde den for infotrygd. Når exception blir kastet, logges det fnr i Kibana, som er en uheldig konsekvens.